frontend/src/components/pages/profile/Profile.tsx

```diff
@@ -11,11 +11,7 @@ import FriendsList from './components/friendsList/FriendsList';
 import Posts from './components/posts/Posts';
 import './profile.scss';
 
-interface MatchParams {
-  id: number & string;
-}
-
-interface IParamsProps extends RouteComponentProps<MatchParams> {}
+interface IParamsProps extends RouteComponentProps<any> {}
```

Comment:
why any?

also this is nonsense type

interface MatchParams {
  id: number & string;
}

should be   id?:  string; (path params are only and always strings | undefined)

Reply:
![image](https://user-images.githubusercontent.com/43353740/99097973-05328b80-25e1-11eb-914d-e135a7e89f88.png)